www/source/partials/docs/_internals-supervisor.html.md.erb

```diff
@@ -6,7 +6,7 @@ Because the basic functionality of process supervision is well-known, this docum
 
 ## Architecture
 
-Supervisors are configured to form a ring by using the `--peer` argument and pointing them at peers that already exist. In a real-life deployment scenario, Supervisors in a ring would also have a shared encryption key, so that inter-Supervisor traffic is encrypted. (See the [security](/docs/glossary/#glossary-keys) documentation for more details.)
+Supervisors are configured to form a ring by using the `--peer` argument and pointing them at peers that already exist. In a real-life deployment scenario, Supervisors in a ring would also have a shared encryption key, so that inter-Supervisor traffic is encrypted. (See the [security](/docs/glossary/#glossary-origin-keys) documentation for more details.)
```

Comment:
```suggestion
Supervisors are configured to form a network by using the `--peer` argument and pointing them at peers that already exist. In a real-life deployment scenario, Supervisors in a network may also have a shared encryption key, so that inter-Supervisor traffic is encrypted. (See the [security](/docs/glossary/#glossary-origin-keys) documentation for more details.)
```

"Ring" means [something specific in the context of networks](https://en.wikipedia.org/wiki/Ring_network), and that's not what we're doing. This term is somewhat pervasive, but as long as we're making a change here, we should remove it.